src/sbvr-api/odata-response.ts

```diff
@@ -158,9 +173,17 @@ export const process = async (
 	);
 
 	const odataIdField = sqlNameToODataName(table.idField);
+	const fetchProcessingPromises: Array<Promise<void>> = [];
```

Comment:
If this is being handled in pinejs like this then I think I'd prefer it be a separate pass like `expandableFields` has where it's done in parallel for only the single field (and only if such a field is included). Obviously if there are benchmarks proving one way or the other to be faster then that'd win but afaik we don't atm

Reply:
I see. With that we can actually stop doing the function that changes the prerespond hook aswell. Will fix it

Reply:
@Page- Fixed it as I think it is what you suggested. The good thing about this approach is that it does not change anything on the request time of requests without a webresource, which I think was your goal here. Let me know what do you think :+1: